### PR TITLE
Log IP on proxy requests

### PR DESCRIPTION
**Pull request by paulgb**

### CODE REVIEW

File: drone/Cargo.lock [hunk 1]
```diff
@@ -1,2745 +0,0 @@
-# This file is automatically @generated by Cargo.
```
Comment (rolyatmax) [inline drone/Cargo.lock:1]: Was deleting this file intentional?

File: dev/tests/proxy.rs [hunk 2]
```diff
@@ -110,6 +110,12 @@ impl Proxy {
             .resolve(&hostname, self.bind_address)
             .build()?;
 
+        let path = if let Some(path) = path.strip_prefix("/") {
+            path
+        } else {
+            path
+        };
+
```
Comment (pretentious7) [inline dev/tests/proxy.rs:118]: I don't think I understand this, why does it return path in both branches?

**Review by pretentious7 — APPROVED** [review]
Lgtm, just a bit confused about that if let

File: dev/tests/proxy.rs (hunk 2 above)
Comment (paulgb) [reply dev/tests/proxy.rs:118]: In the first branch, `path` is shadowed by the `if let Some(path)` assignment. So it's the same variable name in both branches, but they refer to a different value. I probably could have used `unwrap_or` here instead, I'll clean that up next time I see it.

**Pull request merged**